monai/apps/auto3dseg/data_analyzer.py

```diff
@@ -257,7 +252,7 @@ def get_all_case_stats(self, key="training", transform_list=None):
         result[DataStatsKeys.SUMMARY] = summarizer.summarize(result[DataStatsKeys.BY_CASE])
 
         if not self._check_data_uniformity([ImageStatsKeys.SPACING], result):
-            logger.warning("data spacing is not completely uniform. MONAI transforms may provide unexpected result")
+            print("Data spacing is not completely uniform. MONAI transforms may provide unexpected result")
```

Comment:
@myron how about using `logger.info`, if this shouldn't be a warning to the user? 

This check originated from a chat with @dongyang0122 for reasons I don't exactly remember. If this is no longer helpful for the analysis, I am okay not to perform a check of the uniformity.